frontend/apps/case-exploration/src/components/cases/cases-tab.tsx

```diff
@@ -35,32 +35,22 @@ function CasesTab() {
   });
   const [searchCriteria, setSearchCriteria] = useState<SearchCriterion[]>([]);
 
+  const allAdditionalFields = [
```

Comment:
Same define outside component. Not critical but no need to redefine this on every render since its static